src/Module.Server/Common/Models/CrpgAgentStatCalculateModel.cs

```diff
@@ -224,13 +224,13 @@ private void UpdateMountAgentStats(Agent agent, AgentDrivenProperties props)
         int ridingSkill = agent.RiderAgent != null
             ? GetEffectiveSkill(agent.RiderAgent.Character, agent.RiderAgent.Origin, agent.RiderAgent.Formation, DefaultSkills.Riding)
             : 100;
+        int harnessArmor = mountHarness.Item?.ArmorComponent?.BodyArmor ?? 0;
+        float armoredPercentage = harnessArmor / 60f;

```

Comment:
I guess 60 is the max armor, you could move that into a constant
```
const float maxHarnessArmor = 60f;
```

Reply:
done